.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
+
+Each package has its own contributing guide, please follow the links to find the actual contributing guide you might 
+need.
+
+## Dialtone CSS
+[CONTRIBUTING.md](../packages/dialtone/.github/CONTRIBUTING.md)
+
+## Dialtone Tokens
+[CONTRIBUTING.md](../packages/dialtone-tokens/.github/CONTRIBUTING.md)
+
+## Dialtone Vue 3
+[CONTRIBUTING.md](../packages/dialtone-vue3/.github/CONTRIBUTING.md)
```

Comment:
what is packages/dialtone-vue3?? this doesn't exist

Reply:
Oh sorry, I have it on my local because I'm working on that, will remove it for now.